tensorflow/python/keras/layers/local.py

```diff
@@ -729,14 +729,14 @@ def local_conv_matmul(inputs, kernel, kernel_mask, output_shape):
   Returns:
       Output (N+2)-D tensor with shape `output_shape`.
   """
-  inputs_flat = backend.reshape(inputs, (backend.shape(inputs)[0], -1))
+  inputs_flat = array_ops.reshape(inputs, (array_ops.shape(inputs)[0], -1))
```

Comment:
I'm wondering if we can somehow mentioned on why there's changes on the implementation on the PR's description, previously we're using `backend` instead of `array_ops`.

Reply:
@irvifa 
Can you please elaborate on that?

Reply:
Actually in the issue the user pointed out that some of the calls were deprecated. Also, backend also calls `array_ops` methods, so I called them directly.